Update interprocess.md (#30310)
dotnet/AspNetCore.Docs: aspnetcore/grpc/interprocess.md

## Inter-process communication (IPC) transports

> gRPC calls between a client and server on different machines are usually sent over TCP sockets. TCP is a great choice for communicating across a network or the Internet. However, IPC transports offer advantages when communicating between processes on the same machine:

Reviewer: ```suggestion
gRPC calls between a client and server on different machines are usually sent over TCP sockets. TCP is a good choice for communicating across a network or the Internet. However, IPC transports offer advantages when communicating between processes on the same machine:
```

Reviewer: Technical document, not marketing doc.

* Less overhead and faster transfer speeds.
* Integration with OS security features.
* Doesn't use TCP ports, which are a limited resource.

.NET supports multiple IPC transports:

* [Unix domain sockets (UDS)](https://wikipedia.org/wiki/Unix_domain_socket) is a widely supported IPC technology. UDS is the best choice for building cross-platform apps, and it's usable on Linux, macOS, and [Windows 10/Windows Server 2019 or later](https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/).
* [Named pipes](https://wikipedia.org/wiki/Named_pipe) are supported by all versions of Windows. Named pipes integrate well with [Windows security](/windows/win32/ipc/named-pipe-security-and-access-rights), which can control client access to the pipe.

> * Support other IPC transports by implementing <xref:Microsoft.AspNetCore.Connections.IConnectionListenerFactory> and registering the implementation at app startup.

Reviewer: ```suggestion
* Additional IPC transports by implementing <xref:Microsoft.AspNetCore.Connections.IConnectionListenerFactory> and registering the implementation at app startup.
```

Reviewer: Not really happy with that last bullet. Is there a better adjective than Additional. Something more like custom.

> Depending on the current OS, cross-platform apps may want to use different IPC transports. An app can check the current operating system on startup and choose the desired transport for that platform:

Reviewer: I get why you're using current, so you can determine the OS at run time. But I don't think current is needed. Our mantra, less is more. [Use bigger ideas, fewer words](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#use-bigger-ideas-fewer-words) : Our modern design hinges on crisp minimalism. Shorter is always better.
[Short sentences and fragments are easier to scan and read.](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human#a-focus-on-the-customer) Prune every excess word.

Author: ```suggestion
Depending on the OS, cross-platform apps may want to use different IPC transports. An app can check the operating system on startup and choose the desired transport for that platform:
```

> Depending on the current OS, cross-platform apps may want to use different IPC transports. An app can check the current operating system on startup and choose the desired transport for that platform:

Reviewer: ```suggestion
Depending on the OS, cross-platform apps may choose different IPC transports. An app can check the OS on startup and choose the desired transport for that platform:
```
